setup.py

```diff
@@ -60,4 +60,8 @@
         'scipy == 0.17.1',
     ],
 
+    # pytest integration
+    setup_requires=['pytest-runner'],
```

Comment:
I think we will have to add the pytest version to the install_requires variable. Was not sure about and was hoping someone will comment :)